week-1/3-extra/1-currency-conversion.js

```diff
@@ -16,7 +18,9 @@ function convertToUSD() {}
   Find a way to add 1% to all currency conversions (think about the DRY principle)
```

Comment:
I'm going to guess what they mean here about DRY - (Don't repeat yourself)...
Rather than hard coding sums into each function you could add some lower-level functions that can be reused.
eg:
```
function multiply(x, y) {
  return x * y
}

function add(x, y) {
   return x + y
}

function divide(x, y) {
   return x / y
}

function getPercentage(full, portion) {
  return multiply( divide(full, portion), 100 )
}

function getExchange(value, rate) {
  return multiply(value, rate)
}

function convertToUSD(price) {
  return getExchange(price, 1.4)
}

function convertToBRL(price) {
  const exchange = getExchange(price, 5.7)
  return add(exchange, getPercentage(1, 5.7))
}
```

This seems like a lot of code for a something so simple but it can be a lot easier to read - and also - if you were building a currency conversion app - you can see how these functions might be reused over and over again.